opencog/persist/zmq/atomspace/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ ADD_CUSTOM_COMMAND(
 )
 
 ADD_LIBRARY (zmqatoms SHARED
-	ZMQMessages.pb
+	ZMQMessages.pb.cc ZMQMessages.pb.h
```

Comment:
Yes, without .cc extension, CMake silently ignores ZMQMessages.pb.cc (verified by looking at 'make VERBOSE=1' output) and I got many 'undefined reference' errors for functions/variables defined in this file when trying to link opencog to this library. I don't know why (at least, it should print error message about not finding the source file). But, adding .cc extension explicitly fixes the problem I have.

(more commits will be added to this branch soon, since there are other problems too).